docs/2_managing-patched-applications.md

```diff
@@ -0,0 +1,13 @@
+# 🧰 Managing patched applications
+1. Tap on the **Dashboard** tab in the bottom navigation bar
+2. Select the **Installed** chip
+3. You will see a list of all apps patched by the Manager. Tap on the **Info** button for the application you want to manage.
+
+Here, you will be greeted with buttons to open, uninstall, or repatch the application. You will also see information regarding the app, such as when it was patched, the type of installation, and the selected patches. For rooted installations, you'll see the unpatch option instead of uninstall, this functions similarly to the uninstall button.
+
+ > **Note**: Updating patched apps is currently removed and will be added back at a later date.
+
+## ⏭️ What's next
+The next section will inform you on how to update ReVanced Manager.
+
+Continue: [🔄 Updating ReVanced Manager](3_updating-revanced-manager.md)
```

Comment:
```suggestion
# 🧰 Managing patched applications

1. Tap on the **Dashboard** tab in the bottom navigation bar
2. Select the **Installed** chip
3. You will see a list of all apps patched by the Manager. Tap on the **Info** button for the application you want to manage.

Here, you will be greeted with buttons to open, uninstall, or repatch the application. You will also see information regarding the app, such as when it was patched, the type of installation, and the selected patches. For rooted installations, you'll see the unpatch option instead of uninstall, this functions similarly to the uninstall button.

 > **Note**: Updating patched apps is currently removed and will be added back at a later date.

## ⏭️ What's next

The next section will inform you on how to update ReVanced Manager.

Continue: [🔄 Updating ReVanced Manager](3_updating-revanced-manager.md)
```